### ISSUE TITLE
Missile

### ISSUE THREAD

**Issue by alikeobservant** [studio-minus/ppg-bugs-and-requests#503]
**Feature description**
Spawns an explosive missile. Pressing F on it will ignite the thruster, and upon impacting something, it explodes.
**Feature purpose**
Would make things easier, such as a creating a missile truck
**Additional context**
I'm aware things like these have to be created by the player, but making something as simple as a missile that doesn't randomly detonate can be very challenging. This is just so people who aren't skilled at making things like that and just want a missile to play around with, can have one, with the option of creating their own still open.

**Comment by mestiez** [comment]
> making something as simple as a missile that doesn't randomly detonate can be very challenging

this is valid but it can also be used to justify the addition of literally any common complex object. i am unsure if a missile is common enough & complex enough to be worth the clutter.

**Comment by alikeobservant** [comment]
I agree, but I have seen numerous mods where they include something like a missile launcher, e.g submarines, and the missiles always require some personal tweaking after spawning to make them function, and I think with its addition it'd be very helpful for mod creators and sandboxers alike.
